Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs

```diff
@@ -112,7 +112,7 @@ protected virtual void Awake()
 
         protected virtual void Update()
         {
-            if (controllerEvents != null)
+            if (controllerEvents != null && controllerEvents.touchpadTouched && controllerEvents.touchpadAxisChanged)
```

Comment:
doing the ` && controllerEvents.touchpadAxisChanged` causes the pointer direction indicator to not update it's position when you release and then re-touch the touchpad which is incorrect behaviour